packages/eslint-plugin/docs/rules/only-throw-error.mdx

```diff
@@ -102,6 +102,27 @@ This rule adds the following options:
 
 ```ts
 interface Options {
+  /**
+   * Type specifiers that can be thrown.
+   */
+  allowThrowing?: (
```

Comment:
🤔 I'd lean towards `allow` as the option name. IMO it's pretty clear from the rule's name what _"allow"_ would refer to here. Was there a strong reason to differ from #9525 and go with _"allow throwing"_?